data/human/human missions.txt

```diff
@@ -5547,3 +5547,228 @@ mission "Quicksilver Mixup 1"
+					to display
+						has "Blind Man from Martini: label blind"
+						not "Blind Man from Martini: label atomic-bombings"
+
+				`	"Was your daughter caught in the atomic bombings?"`
+					goto daughter
+					to display
+						or
+							has "Blind Man from Martini: label blind"
+							has "Blind Man from Martini: label illness"
+						not "Blind Man from Martini: label daughter"
+				
+				`	"What happened to your daughter?"`
+					goto illness
+					to display
+						not "Blind Man from Martini: label illness"
+						has "Blind Man from Martini: label daughter"
+
+				`	(Stay silent.)`
+					goto waiting
+					to display
+						not "Blind Man from Martini: label waiting"
+
+			label blind
+			action
+				set "Blind Man from Martini: label blind"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"It was the fourth of July, 3014. I was in my office, on one of the high rises on the outskirts of the capital, with my office window pointed right at the city center." He turns to look out the window, as if he knew New Sydney's skyline was out there where he couldn't see. In the small lull, he takes an audible breath.`
+			`	"It was a beautiful day that day. I... I remember that Pollux was especially bright and colorful that day, and the songbirds that inhabit the city were alive with sound. They were unusually so. It was very vivid, very strange. Maybe I should have listened to the birds...`
+			`	"But when the bomb dropped, I was looking right at it, from my office window. All I could remember next was that I was in a lot of pain, that I couldn't see, but my secretary - thank goodness for her - found me afterwards, during the evacuation, covered in glass from the shockwave. I survived mostly because I was sitting at my terminal and that protected me from the bulk of the shards.`
+				goto taxi
+
+			label blind-1
+			action
+				set "Blind Man from Martini: label blind-1"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. I took my life for granted then, but anything can be taken away from you if you get too used to it."`
+				goto taxi
+			
+			label atomic-bombings
+			action
+				set "Blind Man from Martini: label atomic-bombings"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. I was lucky, I could get treatment for my blindness. But others, no...`
```

Comment:
Did the treatment he got actually do anything?

Reply:
I was thinking that he could live by himself fairly well, instead of needing a caretaker or a guide dog or someone like that. But if the treatment's effects should be spelled out, I'm happy to find something or hear suggestions. The main point was that people were affected by the bombings, and I read that permanent blindness from the flash of the explosion was possible.

The mission was partly inspired by a documentary on Al Jazeera about a Polish man who had incurable blindness; during filing he could still barely see, but his eyesight was getting worse and he relied on his son. The name of the documentary is a little ironic for ES: "A Sky Without Stars". 

While the source of Douglas' blindness is different to the Polish man's, I'd think that in ES's universe the both Douglas and the Polish man's blindness could be cured. What the treatment means, I don't really know.

Reply:
```suggestion
			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. In my case, I wanted treatment and received it. But others, no...`
```

Reply:
I really appreciate that you took the time to ask someone about this! I would commit this suggestion (I understand the change), but the purpose of that line was to highlight that there are other people blinded by the explosion who could not get treatment because of the culture in the Paradise Worlds. For example, there's talk of servants and people that are indentured servants in all but name in the Paradise Worlds. I don't imagine that they would be allowed treatment because they couldn't afford it, or because of the breaking down of the health system due to the bombings. Douglas becomes aware of this because he's suffered something similar, but as he's in a better position than them he can get a faster or better treatment; he's grown to realise that he was probably too selfish then in jumping the line. Just some thoughts.

Reply:
Happy to do it, and appreciate the openness to suggestions! I think that's a fair point. The person I spoke to discussed the idea of saying he's "fortunate" as a way of negatively viewing blindness, which could be rude to people who were born that way. Since Douglas isn't born blind and instead becomes blind after experiencing sight, that's a bit different (and this is where the person I consulted was a little unsure herself). So up to you, I'd say. I think there's arguments for both lines on this, so a bit of a toss-up.

Reply:
He talks about needing help to "just live," and it seems to me like he's speaking from personal experience (that is to say, *he* himself needed that help), so it seems a little strange that he goes on to talk about treatment he was able to get, because he doesn't actually give any indication of how that treatment helped, what difference it made, and why others may have had a harder time without it.

Orientation and mobility training is a real life thing that may be relevant: helps to provide blind/visually impaired people with skills to aid them in navigating without sight. For example, cane training.
There's also vocational rehabilitation, which helps provide other skills. In general, it can help provide blind/VI people with the sorts of skills needed to "just live."

Having or lacking access to these sorts of things would certainly make a huge difference for people in Douglas' situation.

Reply:
Ok, this is a good question. I might think about adding some of this in another PR now that it's already been merged, maybe with roadrunner's suggestion too.